<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountConnection</class>
 <widget class="QDialog" name="AccountConnection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>275</width>
    <height>213</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <item row="1" column="0">
    <widget class="QLabel" name="loginLabel">
     <property name="text">
      <string>login</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="loginLineEdit"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="passwordLabel">
     <property name="text">
      <string>password</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="passwordLineEdit"/>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="serverIpLineEdit"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="serverIpLabel">
     <property name="text">
      <string>Server Ip</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="serverPortLineEdit"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="serverPortLabel">
     <property name="text">
      <string>Server Port</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
